iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/pipe/AbstractOperatePipeProcedureV2.java

```diff
@@ -224,7 +224,7 @@ protected Flow executeFromState(ConfigNodeProcedureEnv env, OperatePipeTaskState
     try {
       switch (state) {
         case VALIDATE_TASK:
-          if (executeFromValidateTask(env)) {
+          if (!executeFromValidateTask(env)) {
```

Comment:
The corresponding comment in AbstractOperatePipeProcedureV2 should also be modified.